third_party/chromium/media/METADATA

```diff
@@ -5,9 +5,9 @@ description:
 third_party {
   url {
     type: GIT
-    value: "https://chromium.googlesource.com/chromium/src/media"
+    value: "https://chromium.googlesource.com/chromium/src"
   }
-  version: "4dc2af87b3a5ff1c1489ec642491b59e9872ac8b"
+  version: "96.0.4664.208"
```

Comment:
These two commits have the same `media` tree ff50d0df:
https://chromium.googlesource.com/chromium/src/+/4dc2af87/media
https://chromium.googlesource.com/chromium/src/+/96.0.4664.208/media

Using the release tag lets us see where we are with respect to Chromium.
Incidentally this tree should probably be updated from the last LTS-96
release 96.0.4664.219.
https://chromereleases.googleblog.com/2022/08/long-term-support-channel-update-for_31.html